grapher/update.php

```diff
@@ -49,12 +49,11 @@
 
 echo "Latest Version: $latest<br><br>";
 
-if(isset($_POST['password']) || isset($_POST['yup']) || isset($_GET['password'])){
```

Comment:
the get['password'] is there for me to easily update multiple servers... and the yup is there so it doesn't always update when checking if latest version on windows app.